docs/index.rst

```diff
@@ -46,5 +72,42 @@ reuse already existing properties file since no changes were introduced there)
 
     pytest tests/foreman/ui_airgun/test_architecture.py
 
+
+Contributing
+------------
+
+Good practice when working with Python project as AirGun is to work in
+Python's virtual environments. To initiate and activate them before running
+`pip ...` commands run:
+
+.. code-block:: bash
+
+    virtualenv venv   # on Fedora you might need to use `virtualenv-3` if you have Python 2 as a default on your system
+    source venv/bin/activate
+
+If you are about to start contributing, please follow `GitHub flow`_.
+
+For a new-comer, we have issues tagged with "`good first issue`_" so that
+might be place where you can start?
+
+When testing your AirGun code using Robottelo tests, I have used following
+ugly trick to have mine current development version of AirGun available in
+Robottelo virtual environment:
```

Comment:
i'd rather use PYTHONPATH than such symlink - `export PYTHONPATH="$PYTHONPATH:/path/to/airgun/"` and would probably specify that in .bashrc/.zshrc
Shouldn't be an issue when we publish airgun to pypi.
And please don't use words like `ugly trick` in official documentation 🙂